discord/enums.py

```diff
@@ -508,6 +508,16 @@ class NSFWLevel(Enum):
     safe = 2
     age_restricted = 3
 
+class TimeMentionType(Enum):
+    default         = "f"
+    short_date_time = "f"
+    long_date_time  = "F"
+    short_date      = "d"
+    long_date       = "D"
+    short_time      = "t"
+    long_time       = "T"
+    relative        = "R"
+
```

Comment:
I'm not sure this needs to be an Enum, just type hinting the argument with `typing.Literal` probably suffices just file.

Either way unsure if `Mention` if the correct name for this, it's not neccisarrily a mention. just a markdown represenation of a unix timestamp.

Api Docs call this a style so perhaps

`UnixTimestampStyle = Literal["f", "F", "d", "D", "t", "T", "R"]` 

in utils.py would be better suited. can just link to the API docs for what each style does anyway,

Reply:
I disagree, an enum seems to be more fitting for this, and just using `Literal` wouldn't really fit in with the rest of the lib